### ISSUE TITLE
Failing instrumented test doesn't fail build

### ISSUE THREAD

**Issue by vitusortner** [Malinskiy/action-android#13]
Failing instrumented tests don't fail our GitHub Actions build. The used workflow can be found in the following. Is it a configuration error?

```yaml
name: CI

on: push

jobs:
  instrumented:
    name: Instrumented tests
    runs-on: macOS-10.14
    env:
      SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
      SIGNING_STORE_PASSWORD: ${{ secrets.SIGNING_STORE_PASSWORD }}
    strategy:
      matrix:
        android-api: [21, 24]

    steps:
      - uses: actions/checkout@v1
      - name: Install Android SDK
        uses: malinskiy/action-android/install-sdk@release/0.0.3
      - name: Assemble APK
        run: ./gradlew assembleStagingDebug
      - name: Run instrumented tests
        uses: malinskiy/action-android/emulator-run-cmd@release/0.0.3
        with:
          cmd: ./gradlew connectedStagingDebugAndroidTest
          api: ${{ matrix.android-api }}
          tag: default
          abi: x86
```

Partly log of step "Run instrumented tests":
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:connectedStagingDebugAndroidTest'.
> There were failing tests. See the report at: file:///Users/runner/runners/2.161.0/work/foo-android-v2/foo-android-v2/app/build/reports/androidTests/connected/flavors/STAGING/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 59s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.4.1/userguide/command_line_interface.html#sec:command_line_warnings
60 actionable tasks: 39 executed, 21 up-to-date
stopping emulator
bash -c \/Users/runner/android-sdk/platform-tools/adb -s emulator-5554 emu kill
OK: killing emulator, bye bye
OK
emu kill finished
emulator is stopped
emulator: WARNING: Discarding the changed state (command-line flag).
```

**Comment by Malinskiy** [comment]
Hey @vitusortner. Let me verify this today and I'll get back to you

**Comment by Malinskiy** [comment]
fix released in 0.0.5
please reopen if your issue persists

**Comment by vitusortner** [comment]
Works now. Thanks for the quick fix!
